R/construct_design.R

```diff
@@ -128,31 +119,19 @@
 construct_design <- function(...) {
   
   qs <- quos(...)
+  qs <- maybe_add_names_qs(qs)
```

Comment:
note that you don't really use qnames now, can probably purge this and set names(ret) directly.

Reply:
we do that a bit lower, but need here to capture names like pop when sent to `pop + declare_sampling(n = 50)`. the name `pop` is lost if you don't use maybe_add_names on the quosures.

Reply:
There's only one place that calls in to conduct_design, should not need any NSE within it - pass the steps directly. See #249